KnowSheet/scripts/coverage-report.sh

```diff
@@ -7,23 +7,26 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -fprofile-arcs -ftest-coverage -DBRICKS_COVERAGE_REPORT_MODE"
 LDFLAGS="-pthread"
 
-TMPDIR=.noshit
+SCRIPT_DIR=$( $( dirname "${BASH_SOURCE[0]}" )/KnowSheetReadlinkDir.sh "$PWD" )
 
-mkdir -p $TMPDIR
+TMPDIR=".noshit"
+TMPDIR_FULL_PATH="$SCRIPT_DIR/.noshit"
```

Comment:
This directory should be relative to the directory where the script is run from, ex. `net/api`, not the `coverage-report.sh` script, which is `Bricks/KnowSheet/scripts/`.